eng/imports/Versions.props

```diff
@@ -0,0 +1,6 @@
+<!-- Any changes to this file or format requires updates in project-system-vscode -->
+<Project>
+  <PropertyGroup>
+    <CPSPackageVersion>17.7.294-pre</CPSPackageVersion>
```

Comment:
Why is this here and not in any of the other props files?

Reply:
This file is used in project-system-vscode which adds element for PS.Managed version into it and uses it in other repo. It is separate since it is being used by automation. We can add more items to this file but to keep automation simple, a simple separate file like this is needed.